lib/tasks/import_bounding_boxes.rake

```diff
@@ -20,4 +20,31 @@ namespace :import do
       end
     end
   end
+
+  def import_bounding_boxes_from_esri
+    filename = "lib/data/habitat_presence/habitat_presence_country.csv"
```

Comment:
I've always seen Rails.root used for pathnames, not sure if it's needed. Will this work on staging and production?

Reply:
Yes, it should work on staging and production (and it has done actually since this functionality to import the bounding boxes is already on staging).